### PR TITLE
Update document and add release note for helpshift-plugin-sdkx-react-native-10.2.4.

### PR DESCRIPTION
**Pull request by iosdevhelpshift**
* Added release note for 10.2.4

### CODE REVIEW

File: docs/sdkx-react-native/release-notes-react-native.mdx [hunk 1]
```diff
@@ -23,6 +23,14 @@ import {
 
 ## 10.2.3
 
+_June 2nd, 2023_
+
+** Release Notes: **
+
+- Added an API to pause/resume the In-app notifications. For more details, refer our [Notifications](/sdkx-react-native/notifications-ios/#pause-in-app) page.
```
Comment (sumeet-helpshift) [inline docs/sdkx-react-native/release-notes-react-native.mdx:30]: We can just mention like this:
For more details, refer: [Notifications](/sdkx-react-native/notifications-ios/#pause-in-app).

Comment (iosdevhelpshift) [reply docs/sdkx-react-native/release-notes-react-native.mdx]: @sumeet-helpshift  updated.

**Review by sumeet-helpshift — APPROVED** [review]

File: docs/sdkx-react-native/notifications-ios.mdx [hunk 2]
```diff
@@ -237,6 +237,20 @@ If you do not want the in-app notifications to support provided by the
 Helpshift SDK, please set this flag to `false`. The default value of this
 flag is `true` i.e in-app notifications will be enabled.<br />
 
+### Pausing In-app notifications {#pause-in-app}
+
+If you have enabled in-app notifications, use the API `pauseDisplayOfInAppNotification:` to pause/resume the notifications. When `true` is passed to this method display of in-app notifications is paused even if they arrive. When you pass a `false`, the in-app notifications start displaying.
```
Comment (jaydeep-helpshift) [inline docs/sdkx-react-native/notifications-ios.mdx:242]: - Why is there a `:` after API name on this line?
- When `true` is passed to this method, display of in-app notifications....
- When you pass `false`, the in-app.....

Fix in other pages as well if needed.

File: docs/sdkx-react-native/release-notes-react-native.mdx [hunk 3]
```diff
@@ -21,6 +21,14 @@ import {
 
 # Release Notes
 
+## 10.2.4
+
+_June 2nd, 2023_
+
+** Release Notes: **
+
+- Added an API to pause/resume the In-app notifications. For more details, refer:[Notifications](/sdkx-react-native/notifications-ios/#pause-in-app) page.
```
Comment (jaydeep-helpshift) [inline docs/sdkx-react-native/release-notes-react-native.mdx:30]: For more details, refer [Notifications](url) page.

**Review by jaydeep-helpshift — CHANGES_REQUESTED** [review]

File: docs/sdkx-react-native/release-notes-react-native.mdx (hunk 3 above)
Comment (iosdevhelpshift) [reply docs/sdkx-react-native/release-notes-react-native.mdx:30]: Updated.

**Review by jaydeep-helpshift — APPROVED** [review]

**Pull request merged**